scripts/travis/build.sh

```diff
@@ -54,24 +66,36 @@ cd $build_path
 #
 progress "Configuring with cmake"
 
-cmake_flags="-DARB_WITH_ASSERTIONS=on -DARB_WITH_MPI=${WITH_MPI} ${CXX_FLAGS}"
+cmake_flags="-DARB_WITH_ASSERTIONS=ON -DARB_WITH_MPI=${WITH_MPI} -DARB_WITH_PYTHON=${ARB_WITH_PYTHON} ${CXX_FLAGS}"
 echo "cmake flags: ${cmake_flags}"
 cmake .. ${cmake_flags} || error "unable to configure cmake"
 
 export NMC_NUM_THREADS=2
 
-progress "Unit tests"
+progress "C++ unit tests"
 make unit -j4                || error "building unit tests"
 ./bin/unit --gtest_color=no  || error "running unit tests"
 
-progress "Distributed unit tests (local)"
+progress "C++ distributed unit tests (local)"
 make unit-local -j4          || error "building local distributed unit tests"
 ./bin/unit-local             || error "running local distributed unit tests"
 
-if [[ "${WITH_DISTRIBUTED}" = "mpi" ]]; then
-    progress "Distributed unit tests (MPI)"
+if [[ "${WITH_DISTRIBUTED}" == "mpi" ]]; then
+    progress "C++ distributed unit tests (MPI)"
     make unit-mpi -j4        || error "building MPI distributed unit tests"
     ${launch} ./bin/unit-mpi || error "running MPI distributed unit tests"
 fi
 
+if [[ "${WITH_PYTHON}" == "true" ]]; then
+    progress "Building python module"
+    make pyarb -j4                                                           || error "building pyarb"
+    if [[ "${WITH_DISTRIBUTED}" == "serial" ]]; then
```

Comment:
Should we always run the serial unit tests for Python, just like we always run the serial C++ tests?

Reply:
Since I included the `WITH_PYTHON=true` in the `.travis.yml` for all entries in the matrix, I would leave it here. Basically, that way we already always run the serial unit test. I just introduced `WITH_PYTHON` in case we would like to deactivate python for mpi testing. Then we only have to set it to false in the mpi matrix entries.

Reply:
This looks like it won't run the serial test if `WITH_PYTHON==true` and `WITH_DISTRIBUTED=mpi`.

Reply:
Yes, you`re right. I'll change that.

Reply:
Done.